packages/components/src/badge/README.md

```diff
@@ -20,3 +20,7 @@ Badge variant.
  - Type: `"default" | "info" | "success" | "warning" | "error"`
  - Required: No
  - Default: `default`
+
+### `Additional Props`
```

Comment:
@mirka, didn't we use an auto-generated README for the new Badge component? 😭

Reply:
It's ok, a CI check will fail when this happens, so someone will notice one way or another before merge 😄